Sources/SmileID/Classes/Networking/Models/Services.swift

```diff
@@ -0,0 +1,100 @@
+import Foundation
+
+// MARK: - ServicesResponse
+
+public struct ServicesResponse: Codable {
+    let bankCodes: [BankCode]
+    let hostedWeb: HostedWeb
+
+    enum CodingKeys: String, CodingKey {
+        case bankCodes = "bank_codes"
+        case hostedWeb = "hosted_web"
+    }
+}
+
+// MARK: - BankCode
+
+public struct BankCode: Codable {
+    let name, code: String
+}
+
+// MARK: - HostedWeb
+
+public struct HostedWeb: Codable {
+    let basicKyc: CountryCodeToCountryInfo
+    let biometricKyc: CountryCodeToCountryInfo
+    let enhancedKyc: CountryCodeToCountryInfo
+    let docVerification: CountryCodeToCountryInfo
+    let enhancedKycSmartSelfie: CountryCodeToCountryInfo
+
+    private enum CodingKeys: String, CodingKey {
+        case basicKyc = "basic_kyc"
+        case biometricKyc = "biometric_kyc"
+        case enhancedKyc = "enhanced_kyc"
+        case docVerification = "doc_verification"
+        case enhancedKycSmartSelfie = "ekyc_smartselfie"
+    }
+}
+
+// MARK: - CountryInfo
+
+public struct CountryInfo: Codable {
+    let countryCode: String
```

Comment:
countryCode isn't actually returned by the api in this object. It was added on Android as a convenience, and has to be manually set.

1. Can you add a comment indicating this
2. How will the deserializer handle this?/How will this value actually get set?